src/writer/sync.py

```
@@ -248,36 +248,48 @@
                 if self.operation_queues.dirs_to_create:
                     self.log(f"Creating {len(self.operation_queues.dirs_to_create)} directories")
                     for dir_path in self.operation_queues.dirs_to_create.copy():
-                        to = self.dest_path(dir_path)
-                        self.ensure_dir(to)
-                        self.log("Dir added:", self.rel(dir_path))
+                        try:
+                            self.log("Dir add:", self.rel(dir_path))
+                            to = self.dest_path(dir_path)
+                            self.ensure_dir(to)
+                        except Exception as e:
```

Pylint (3.3.7)

[warning] Catching too general exception Exception

(W0718)